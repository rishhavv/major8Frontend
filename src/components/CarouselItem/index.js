import {Image} from "antd";
import React from "react";

const styles={
    background:{
        backgroundSize: 'cover',
        backgroundRepeat: 'no-repeat',
        width:'100%',
        height:'auto',
    }
}

export const CarouselItem=({url})=>{
    return(
        <div style={styles.background}> 
            <Image src={url}></Image>
        </div>
        
    )
}

export default CarouselItem