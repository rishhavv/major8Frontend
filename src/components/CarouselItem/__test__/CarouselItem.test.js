import {cleanup, fireEvent, render} from '@testing-library/react';
import CarouselItem from '../index.js'
import renderer from 'react-test-renderer'

import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

configure({adapter: new Adapter()});

afterEach(cleanup)


    const url="assets/post/11.jpeg"

    it('Should render image correctly',()=>{
        const wrapper=shallow(
            <CarouselItem
            url={url}
            />
        )
    expect(wrapper).toMatchSnapshot()
    })
