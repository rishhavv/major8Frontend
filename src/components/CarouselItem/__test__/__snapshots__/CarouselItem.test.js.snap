// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render image correctly 1`] = `ShallowWrapper {}`;
